### ISSUE TITLE
Improvements to testing workflow

### ISSUE THREAD

**Issue by vteague** [DemocracyDevelopers/colorado-rla#219]
The workflow tests are working individually, but there are some improvements that would be good.

1. Make sure they can run in parallel - at the moment they seem to be clashing. Perhaps putting a separate port number into the testclass-specific properties file would fix it.
2. Figure out how to get them working in the github actions flow. (Possibly fixing the above will do this; possibly there are other reasons as well.)

**Comment by vteague** [comment]
This is superseded by @michelleblom 's improved workflows.
